### PR TITLE
QA8-1471 add negative tests for image

### PR DESCRIPTION
**Pull request by Mykhaylyak**

### CODE REVIEW

File: test/extended/image.js [hunk 1]
```diff
@@ -116,4 +116,49 @@ describe('Image field suit', function () {
             expect(imagesList).toBeDisplayed();
         });
     });
+
+    describe('Negative cases', function () {
+
+        it('TC-140 Verify that  size JPEG images 4 Mb by clicking', function () {
+            uploadImage(images.["JPG4Mb"]);
```
Comment (denispalkin) [inline test/extended/image.js:123]:
```suggestion
            uploadImage(images["JPG4Mb"]);
```

File: test/extended/image.js [hunk 2]
```diff
@@ -116,4 +116,49 @@ describe('Image field suit', function () {
...
+            let imageError = $(sel.imageError).waitForDisplayed(100);
+            expect(imageError).toEqual(true);
+        });
+
+        it('TC-142 Verify that size PNG images 4 Mb by clicking', function () {
+            uploadImage(images.["PNG4Mb"]);
```
Comment (denispalkin) [inline test/extended/image.js:129]:
```suggestion
            uploadImage(images["PNG4Mb"]);
```

File: test/extended/image.js [hunk 3]
```diff
@@ -116,4 +116,49 @@ describe('Image field suit', function () {
...
+            let imageError = $(sel.imageError).waitForDisplayed(100);
+            expect(imageError).toEqual(true);
+        });
+
+        it('TC-145 Verify that user can upload a HEIC file by clicking the Image upload area ', function () {
+            uploadImage(images.["HEIC2Mb"]);
```
Comment (denispalkin) [inline test/extended/image.js:135]:
```suggestion
            uploadImage(images["HEIC2Mb"]);
```

File: test/extended/image.js [hunk 4]
```diff
@@ -116,4 +116,49 @@ describe('Image field suit', function () {
...
+            let imageError = $(sel.imageError).waitForDisplayed(100);
+            expect(imageError).toEqual(true);
+        });
+
+        it('TC-147 Verify that user can upload a GIF file by clicking the Image upload area ', function () {
+            uploadImage(images.["GIF2Mb"]);
```
Comment (denispalkin) [inline test/extended/image.js:141]:
```suggestion
            uploadImage(images["GIF2Mb"]);
```

File: test/extended/image.js [hunk 5]
```diff
@@ -116,4 +116,49 @@ describe('Image field suit', function () {
...
+            let imageError = $(sel.imageError).waitForDisplayed(100);
+            expect(imageError).toEqual(true);
+        });
+
+        it('TC-149 Verify that user can upload a MP4 file by clicking the Image upload area ', function () {
+            uploadImage(images.["MP4_2Mb"]);
```
Comment (denispalkin) [inline test/extended/image.js:147]:
```suggestion
            uploadImage(images["MP4_2Mb"]);
```

File: test/extended/image.js [hunk 6]
```diff
@@ -116,4 +116,49 @@ describe('Image field suit', function () {
...
+            let imageError = $(sel.imageError).waitForDisplayed(100);
+            expect(imageError).toEqual(true);
+        });
+
+        it('TC-151 Verify that user can upload a KTX file by clicking the Image upload area ', function () {
+            uploadImage(images.["KTX2Mb"]);
```
Comment (denispalkin) [inline test/extended/image.js:153]:
```suggestion
            uploadImage(images["KTX2Mb"]);
```

File: test/extended/image.js [hunk 7]
```diff
@@ -116,4 +116,49 @@ describe('Image field suit', function () {
...
+            let imageError = $(sel.imageError).waitForDisplayed(100);
+            expect(imageError).toEqual(true);
+        });
+
+        it('TC-153 Verify that user can upload a PDF  file by clicking the Image upload area ', function () {
+            uploadImage(images.["PDF2Mb"]);
```
Comment (denispalkin) [inline test/extended/image.js:159]:
```suggestion
            uploadImage(images["PDF2Mb"]);
```

**Review by denispalkin — DISMISSED** [review]

File: test/extended/image.js (hunk 1 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 2 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 3 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 4 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 5 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 6 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

**Review by dalaran78 — APPROVED** [review]

**Review by Alex-Mashchenko — APPROVED** [review]
Nice

**Review by Alex-Mashchenko — APPROVED** [review]

**Review by MariiaSosedova — APPROVED** [review]

**Review by MariiaSosedova — APPROVED** [review]

**Review by MariiaSosedova — APPROVED** [review]

**Review by denispalkin — APPROVED** [review]
LGTM. Merging!

**Pull request merged**